docs/markdown/snowplow_media_player_col_definitions.md

```diff
@@ -0,0 +1,235 @@
+The type of media content: video or audio.
+{% enddocs %}
+
+{% docs col_media_player_type %}
+The combination of schema_name and schema_vendor coming from the specific media player context. E.g. com.youtube-youtube, org.whatwg-media_element.
+{% enddocs %}
+
+{% docs col_page_referrer %}
+URL of the referrer e.g. `http://www.referrer.com`.
+{% enddocs %}
+
+{% docs col_page_url %}
+The page URL e.g. `http://www.example.com`.
+{% enddocs %}
+
+{% docs col_source_url %}
+The url which shows the source of the media content. For YouTube it is the url context field, for HTML5 it is the source_url field.
+{% enddocs %}
+
+{% docs col_geo_region_name %}
+Visitor region name e.g. `Florida`.
+{% enddocs %}
+
+{% docs col_br_name %}
+Browser name e.g. `Firefox 12`.
+{% enddocs %}
+
+{% docs col_dvce_type %}
+Type of device e.g. `Computer`.
+{% enddocs %}
+
+{% docs col_os_name %}
+Name of operating system e.g. `Android`.
+{% enddocs %}
+
+{% docs col_os_timezone %}
+Client operating system timezone e.g. `Europe/London`.
+{% enddocs %}
+
+{% docs col_duration %}
+Total length of media in seconds.
+{% enddocs %}
+
+{% docs col_playback_rate %}
+Playback rate (1 is normal).
+{% enddocs %}
+
+{% docs col_playback_quality %}
+Depending on the player it is either the playback quality field or the resolution.
+{% enddocs %}
+
+{% docs col_percent_progress %}
+The percent of the way through the media.
+{% enddocs %}
+
+{% docs col_percent_progress_reached %}
+An array of percent progresses reached by the user while playing the media.
+{% enddocs %}
+
+{% docs col_is_muted %}
+If the media element is muted.
+{% enddocs %}
+
+{% docs col_start_tstamp %}
+The `derived_tstamp` denoting the time when the event started.
+{% enddocs %}
+
+{% docs col_end_tstamp %}
+The `derived_tstamp` denoting the time when the last media player event belonging to the specific level of aggregation (e.g.: pageview by media) started.
+{% enddocs %}
+
+{% docs col_play_time_sec %}
+Estimated duration of play in seconds.
+{% enddocs %}
+
+{% docs col_avg_play_time_min %}
+Estimated average duration of plays in minutes.
+{% enddocs %}
+
+{% docs col_avg_play_time_sec %}
+Estimated average duration of plays in seconds.
+{% enddocs %}
+
+{% docs col_first_play %}
+The derived_tstamp of the beginning of the first play of a media element.
+{% enddocs %}
+
+{% docs col_last_play %}
+The derived_tstamp of the beginning of the last play of a media element.
+{% enddocs %}
+
+{% docs col_valid_plays %}
+The sum of all media plays that exceeds the minimum media length set within the variable `snowplow__valid_play_sec`, it is defaulted to 30 (seconds).
+{% enddocs %}
+
+{% docs col_impressions %}
+The number of pageviews where a media content was rendered regardless of whether the media was actually played or not.
+{% enddocs %}
+
+{% docs col_avg_playback_rate %}
+Average playback rate (1 is normal).
+{% enddocs %}
+
+{% docs col_play_rate %}
+Total plays divided by impressions.
```

Comment:
Good point, I'll add this clarification to the definition: `Please note that as the base for media plays is pageview / media_id, in case the same video is played multiple times within the same pageview, it will still count as one play.`